### PR TITLE
Fix lacking space on abbr

### PR DESCRIPTION
**Pull request by zchee**

### CODE REVIEW

File: rplugin/python3/deoplete/sources/deoplete_go.py [hunk 1]
```diff
@@ -79,13 +79,13 @@ def gather_candidates(self, context):
             for complete in result[1]:
                 _class = complete['class']
                 word = complete['name']
-                info = complete['type'].replace('func', '')
+                info = complete['type']
+                _abbr = (word + sep + info).replace(' func', '')
```
Comment (svanharmelen) [inline rplugin/python3/deoplete/sources/deoplete_go.py:83]: Is this space (before the word `func`) intended? `replace(' func', '')`

(just checking)

Comment (zchee) [inline rplugin/python3/deoplete/sources/deoplete_go.py:83]: @svanharmelen Yes, If use `replace('func', '')`

screenshot 2016-04-08 18 13 58

Comment (svanharmelen) [inline rplugin/python3/deoplete/sources/deoplete_go.py:83]: Yeah, just saw it myself as well 👍

**Comment by svanharmelen** [comment]
LGTM! And works as expected... Nice one!

**Comment by zchee** [comment]
@svanharmelen Thanks :tada:

**Pull request merged**